package.json

```diff
@@ -41,6 +42,7 @@
     "helmet": "^4.1.1",
     "hpp": "^0.2.3",
     "http-status-codes": "^2.1.4",
+    "https": "^1.0.0",
```

Comment:
Why do you need that dep?

Reply:
I need that for the httpsAgent containing the certificates.

Reply:
You mean the client side certificate for mTLS (Mutual TLS)?

Reply:
For the the client side certificate. So when I create the axios instance I include a httpsAgent with the public certificate and the private key from the CPO which are needed to send requests to the Hubject server

Reply:
So Hubject enforce usage of mTLS with its CA?

Reply:
yes, exactly

More information under: [https://support.hubject.com/hc/en-us/articles/212080805-1-3-Certificate-Signing-Request-CSR-Generation](url)